packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -29,6 +29,52 @@ const chip = css`
     min-width: 0;
   }
 
+  :host(:not([part~='overflow'])) {
+    padding-inline-end: 0;
+  }
+
+  :host([part~='overflow']) {
+    position: relative;
+    min-width: var(--lumo-size-xxs);
+    margin-inline-start: var(--lumo-space-s);
+  }
+
+  :host([part~='overflow'])::before,
+  :host([part~='overflow'])::after {
+    position: absolute;
+    content: '';
+    width: 3px;
+    height: 21px;
+    border-left: 2px solid;
+    border-radius: 4px 0 0 4px;
+    border-color: var(--lumo-contrast-30pct);
+  }
+
+  :host([part~='overflow'])::before {
+    left: -4px;
+  }
+
+  :host([part~='overflow'])::after {
+    left: -8px;
+  }
+
+  :host([part~='overflow'][label='2']) {
```

Comment:
I don't really like using hardcoded styles for `label='2'` and `label='1'` as that would not work when using native `::part()` - maybe we should add yet another value to the list of `part` instead?

Reply:
True. Do we keep it simple with `::part(one overflow)` and `::part(two overflow)`, or would it be clearer with something like `::part(overflow-1st)` and `::part(overflow-2nd)`?

Reply:
IMO we could use `overflow-one` and `overflow-two` to make it more clear. Short CSS selectors are nice, but it might look weird if someone uses `::part(one)` in a custom theme, without providing additional context.

Reply:
Updated the PR to use `overflow-one` and `overflow-two` part names.